src/gba/apu.cpp

```diff
@@ -365,12 +365,13 @@ void agb_audio_callback(void* _apu, u8 *_stream, int _length)
 	s16* stream = (s16*) _stream;
 	int length = _length/2;
 
-	s16 channel_1_stream[length];
-	s16 channel_2_stream[length];
-	s16 channel_3_stream[length];
-	s16 channel_4_stream[length];
-	s16 dma_a_stream[length];
-	s16 dma_b_stream[length];
```

Comment:
Same as above comments. If MSVC doesn't support Variable Length Arrays, C++ vectors are preferred to using **new** and **delete**.